quantus/metrics/axiomatic_metrics.py

```diff
@@ -386,6 +392,7 @@ def __call__(
                 **self.kwargs,
             )
         a_batch = utils.expand_attribution_channel(a_batch, x_batch_s)
```

Comment:
I wonder if we should abstract away lines 394 and 395 from the metrics call? like have something like def `manage_shapes` where these two lines are called?

Reply:
Hmmm, the second line is not always required, since not all metrics require the axes of 'x' covered by 'a' ...
So I think it may make sense to leave it as is.

Reply:
Sure ok, but please add a single comment above those lines about why these transformations on a_batch is necessary

Reply:
added the comments in the latest commit

Reply:
fantastic, danke schön!